### DISCUSSION TITLE
Pixi.JS / ParticleContainer + GSAP performance on Raspberry PI

### DISCUSSION THREAD

**Discussion by dragoon** [pixijs/pixijs#8487] [Q&A] [answered]
Hi all,

I am animating some objects that I generated myself in PIXI.JS via GSAP.

However, I notice the animation is very slow on Raspberry PI and does not even depend on the number objects. As soon as I animate even a single object, the FPS rate drops to ~30. It also doesn't depend on whether I am using ParticleContainer or not.
According to the docs, ParticleContainer should have a much better performance.

I feel like I've overlooked something, but can't figure it out.

Codepen for testing: https://codepen.io/rprokofyev/pen/OJvRVyJ

**Comment by domis86** [comment]
So with GSAP its 30 FPS but without is 60 FPS?
If you try any anymation without GSAP you get 60 FPS?

**Reply by dragoon** [reply]
I'm a bit new in PIXI, I have only used GSAP to animate things, what else can I use?

**Reply by domis86** [reply]
You dont need any library for simple animation. For example you can increase position x value by +1 on each frame etc - to move some object from left to right on screen.

**Reply by dragoon** [reply]
It's strange, but the FPS rate stays at 60 even if I don't animate anything. But it does go to 60 if I don't call `renderer.render(scene)`, so I suppose PIXI is just doing some heavy stuff there...

**Reply by dragoon** [reply]
Okay, I found something, removing `antialias: true` in renderer constructor allows to get to ~55FPS which is really good.

**Reply by domis86** [reply]
The 60 comes from mechanics of https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame

**Comment by bigtimebuddy** [comment]
Can you confirm that your Raspberry PI is running WebGL? I vaguely remember we had issues with these embedded devices in the past with GPU drivers not working well.

**Reply by dragoon** [reply]
Yes, it's using WebGL, with newer version it's using WebGL by default.

**Reply by bigtimebuddy** [reply]
How does the performance compare with a vanilla Container?

**Reply by dragoon** [reply]
I don't see any difference qualitatively in terms of FPS, but I have not measured precisely.

**Comment by dragoon** [comment, marked as answer]
Removing `antialias: true` in the renderer's constructor allowed to get to ~55FPS on Raspberry PI.
